HearthDb/Card.cs

```diff
@@ -77,6 +79,17 @@ public CardSet Set
 			}
 		}
 
+		private static Dictionary<string, string> SpellstoneStrings = new Dictionary<string, string>() { {"LOOT_043", "GAMEPLAY_AMETHYST_SPELLSTONE_%d" },
+			{ "LOOT_051", "GAMEPLAY_JASPER_SPELLSTONE_%d" },
+			{ "LOOT_064", "GAMEPLAY_SAPPHIRE_SPELLSTONE_%d" },
+			{ "LOOT_091", "GAMEPLAY_PEARL_SPELLSTONE_%d" },
+			{ "LOOT_103", "GAMEPLAY_RUBY_SPELLSTONE_%d" },
+			{ "LOOT_503", "GAMEPLAY_ONYX_SPELLSTONE_%d"},
+			{ "LOOT_507", "GAMEPLAY_DIAMOND_SPELLSTONE_%d"},
+			{ "LOOT_526d", "GAMEPLAY_LOOT_526d_DARKNESS_%d"} };
```

Comment:
You can use the CardIds file for all of these.

It also doesn't look like the value is used for anything in this case? python-hearthstone is probably using this dict for other purposes. You can just make it a List.